alerting/src/test/kotlin/com/amazon/opendistroforelasticsearch/alerting/alerts/AlertIndicesIT.kt

```diff
@@ -33,7 +34,28 @@ class AlertIndicesIT : AlertingRestTestCase() {
         assertIndexExists(AlertIndices.HISTORY_WRITE_INDEX)
     }
 
+    fun `test update alert index mapping with new schema version`() {
+        assertIndexDoesNotExist(AlertIndices.ALERT_INDEX)
+        assertIndexDoesNotExist(AlertIndices.HISTORY_WRITE_INDEX)
+
+        putAlertMappings(AlertIndices.alertMapping().trimStart('{').trimEnd('}')
+                .replace("\"schema_version\": 1", "\"schema_version\": 0"))
+        assertIndexExists(AlertIndices.ALERT_INDEX)
+        assertIndexExists(AlertIndices.HISTORY_WRITE_INDEX)
+        verifyIndexSchemaVersion(AlertIndices.ALERT_INDEX, 0)
+        verifyIndexSchemaVersion(AlertIndices.HISTORY_WRITE_INDEX, 0)
+
+        executeMonitor(createRandomMonitor())
+        Thread.sleep(2000)
```

Comment:
Two things here:
1. We should avoid sleeps in tests as these will add up over time and cause testing to become very long.
2. Since we already executed the monitor an alert should be created?

Or am I understanding the purpose of this sleep wrong?

Reply:
Will remove it.